client/view.js

```diff
@@ -131,6 +125,7 @@ export default class View
 			// someone elses hand
 			this.#playerRows[hand.playerId].querySelector('.js-cards').textContent = hand.cardsRemaining;
 			this.#playerRows[hand.playerId].querySelector('.js-name').textContent = hand.playerName;
+			console.log(hand);

```

Comment:
Cheeky console log has appeared ;)
Always do a `git diff` before a commit and read the diff to see if youre happy submitting every change you see